homework/style.css

```diff
@@ -1,3 +1,118 @@
+body {
```

Comment:
This comment was again somehow lost from my feedback of today, but this is what I also like you to improve: the UI when viewed on a mobile phone:

![image](https://user-images.githubusercontent.com/2788771/66112220-be472980-e5ca-11e9-9c71-e1ef2bddc65c.png)